content/docs/releases/release-notes/release-notes-1.16.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Release 1.16
+description: 'cert-manager release notes: cert-manager 1.16'
+---
+
+cert-manager 1.16 includes various improvements to the metrics in the cert-manager components.
+
+## Breaking changes
+
+1. Venafi Issuer may fail to renew Certificates if the duration conflicts with the CA minimum / maximum duration policy in Venafi.
```

Comment:
When I read this, I thought "where can I know more, how can I remediate this?". I suggest adding a "read more" link.

Another nitpick is the use of slashes. I've read the blog post [And/or: Slash the Slashes](https://oilpatchwriting.wordpress.com/2010/06/15/andor-slash-the-slashes/) and found the author's view on slashes in technical writing interesting:

> Slashes are far too common, and almost always betray a lazy thinker; by yoking two words together with a slash, the writer tells us the words are related, but he or she doesn’t know how. Replace the slash with ‘and’ or ‘or.’

To disambiguate the meaning of the slash, I propose to change "/" to "or".

```suggestion
1. Venafi Issuer may fail to renew certificates if the requested duration conflicts with the CA’s minimum or maximum policy settings in Venafi. For more details, refer to the [Venafi Issuer](#venafi-issuer) section below.
```